Tasks/Banchtchikov/Task4/Task4.py

```diff
@@ -0,0 +1,38 @@
+import os
+os.chdir("F:\Python\Md-PT1-53-22\Tasks\Banchtchikov\Task4")
```

Comment:
I would suggest against the usage of a full path since it will work only on your machine